src/Samplers/adaptive_parzen.jl

```diff
@@ -14,6 +14,9 @@ function adaptive_parzen_normal(
     srtd_mus = []
     sigma = []
     prior_pos = 1
+    # sortperm must be used here because order is applied to unsorted_weights below
+    order = sortperm(mus)
```

Comment:
It is feasible to set `config.linear_forgetting` to zero. Therefore `order` must be created every time, not just where `length(mus) >= 2`.